autoware_adapi_v1_msgs/vehicle/msg/FuelLevel.msg

```diff
@@ -0,0 +1,2 @@
+builtin_interfaces/Time stamp
+float32 fuel_level # fuel level in percentage
```

Comment:
```suggestion
float32 fuel_level # Fuel level in percentage of the total capacity (0.0 - empty, 100.0 - full)
```

While the current comments are sufficient, more detailed comments could provide additional clarity.